### DISCUSSION TITLE
Is there a way to support multiple triggers?

### DISCUSSION THREAD

**Discussion by TreapGoGo** [draivin/hsnips#177] [Q&A]
For example, I tried to write

```hsnips
snippet [times, cross] "times" i
\times
endsnippet

snippet [sufficient, Rightarrow, imply] "sufficient condition" i
\Rightarrow
endsnippet
```

but it failed.

In my opinion, it's not elegant to use regex to implement multiple triggers, because there will be an ugly regex in the picklist. 

It's an even worse idea to copy-paste the code above and change the trigger for many times, because it would be painful if we have to change the body for many times.

I am not sure if there is an easier way to implement multiple triggers. Any help will be appreciated :)

**Comment by jonas-frey** [comment]
You can achieve the desired effect by creating multiple snippets with the same body.

**Comment by XRay71** [comment]
You could define a function in the global scope, then define multiple snippets that call said function to return a value.
